guides/source/action_controller_overview.md

```diff
@@ -1073,10 +1073,10 @@ class ClientsController < ApplicationController
 end
 ```
 
-For this example to work, you have to add the PDF MIME type to Rails. This can be done by adding the following line to the file `config/initializers/mime_types.rb`:
+This example works because [Rails already registers the `"application/pdf"` MIME type](https://github.com/rails/rails/blob/b26a9fd2551a2e65e8deef10f4b5ca6b99d666fb/actionpack/lib/action_dispatch/http/mime_types.rb) at startup (alongside other common types). If you need additional MIME types, call [`Mime::Type.register`](https://api.rubyonrails.org/classes/Mime/Type.html#method-c-register) in the file `config/initializers/mime_types.rb`. For example, this is how you would register the Rich Text Format (RTF):
```

Comment:
I think it's more helpful to document why we need to register the MIME type.
It also helps to document how you can check which MIME types are  already supported.
Maybe something like:
````suggestion
You can call any method on `format` that is an extension registered as a MIME type by Rails.
Rails already registers common MIME types like `"text/html"` and `"application/pdf"`:
```ruby
Mime::Type.lookup_by_extension(:pdf)
# => "application/pdf"
```

If you need additional MIME types, call [`Mime::Type.register`](https://api.rubyonrails.org/classes/Mime/Type.html#method-c-register) in the file `config/initializers/mime_types.rb`. For example, this is how you would register the Rich Text Format (RTF):
````

Reply:
I think that's great as it ties it all together. I see what you meant about the lookup method.